src/planning/testing/planning-test-helper.ts

```diff
@@ -130,7 +130,7 @@ export class PlanningTestHelper extends TestHelper{
       assert.lengthOf(this.suggestions, 1);
       suggestion = this.suggestions[0];
     }
-    this.log(`Accepting suggestion: '${((str) => str.length > 50 ? str.substring(0, Math.min(str.length, 50)).concat('...') : str)(suggestion.descriptionText)}'`);
+    this.log(`Accepting suggestion: '${suggestion.descriptionText ? ((str) => str.length > 50 ? str.substring(0, Math.min(str.length, 50)).concat('...') : str)(suggestion.descriptionText) : suggestion.descriptionText}'`);
```

Comment:
nit - imo to improve readability in this statement:
`${suggestion.descriptionText ? ... : suggestion.descriptionText}` 
the part after `:` should be explicitly "undefined", up to you.

Reply:
Done.